src/igvfd/tests/data/inserts/gene.json

```diff
@@ -0,0 +1,116 @@
+[
+    {
+        "geneid": "912",
+        "uuid": "0c9a9a6f-02de-4072-a5a3-6628f13070f6",
+        "dbxrefs": [
+            "MIM:188410",
+            "HGNC:1637",
+            "ENSEMBL:ENSG00000158473",
+            "IMGT/GENE-DB:CD1D",
+            "Vega:OTTHUMG00000022436",
+            "UniProtKB:P15813"
+        ],
+        "name": "CD1d molecule",
+        "ncbi_entrez_status": "live",
+        "organism": "Homo sapiens",
+        "status": "released",
+        "symbol": "CD1D",
+        "synonyms": [
+            "CD1A",
+            "R3",
+            "R3G1"
+        ]
+    },
+    {
+        "geneid": "2995",
+        "uuid": "83a311d1-7ed3-41bb-85f0-1f00c458e6fc",
+        "dbxrefs": [
+            "MIM:110750",
+            "HGNC:4704",
+            "ENSEMBL:ENSG00000136732",
+            "Vega:OTTHUMG00000131464",
+            "UniProtKB:P04921",
+            "RefSeq:NG_007479"
```

Comment:
Could you add a genecards reference to test the regex

Reply:
Actually we should have one for each regex, so there are a few more I am realizing